server/market/orderrouter.go

```diff
@@ -135,14 +136,19 @@ type FeeSource interface {
 	LastRate(assetID uint32) (feeRate uint64)
 }
 
+type MatchNegotiator interface {
+	AccountStats(acctAddr string, assetID uint32) (qty, swaps uint64, redeems int)
+}
```

Comment:
Maybe define this in balancer.go instead.  None of the other market types care about this.

I think balancer.go could define both this and a `PendingAccounter` that I proposed in another comment.